eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -96,23 +96,34 @@ stages:
                           scriptPath: 'scripts/devops_tasks/verify_dependencies_present.py'
                           arguments: '--package-name ${{ artifact.name }} --service ${{ parameters.ServiceDirectory }}'
 
-                      - task: TwineAuthenticate@1
-                        displayName: 'Authenticate to registry: pypi.org'
-                        inputs:
-                          pythonUploadServiceConnection: 'PyPI Upload (azure-sdk token)'
-
                       - task: TwineAuthenticate@0
                         displayName: 'Authenticate to feed: ${{parameters.DevFeedName}}'
                         inputs:
                           artifactFeeds: ${{parameters.DevFeedName}}
 
-                      - script: |
-                          set -e
-                          twine upload --repository 'pypi' --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*.whl
-                          echo "Uploaded whl to pypi"
-                          twine upload --repository 'pypi' --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*.zip
-                          echo "Uploaded zip to pypi"
-                        displayName: 'Publish package to registry: pypi.org'
+                      - pwsh: |
+                          $esrpDirectory = "$(Pipeline.Workspace)/esrp-release/${{parameters.ArtifactName}}/${{artifact.name}}"
+                          New-Item -ItemType Directory -Force -Path $esrpDirectory
+
+                          Get-ChildItem -Path "$(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}" `
+                            | Where-Object { ($_.Name -like "*.zip" -or $_.Name -like "*.whl") } `
+                            | Copy-Item -Destination $esrpDirectory
```

Comment:
Is a copy actually necessary or can we just pass the existing folder?

Reply:
The folder that we pass in gets a new artifact dropped in it. That new artifact it drops is _also_ a zip which breaks the succeeding step.

So we either add a manual step to copy these things off to a different folder, or a manual step to clean up the new artifact that's dropped. Either way I gotta do something.

I don't want to place the publish to integration above the actual release to PyPI, for consistency reasons.

Reply:
I was just trying to avoid unnecessary copying or deleting and instead just have the uploaded artifact match what we need for publishing without modification when downloaded. I'm sure your remember all the extra filtering we did in the various release steps in the past that is what I'm trying to avoid. However, if that doesn't work with the other ESRP release then copying should be fine.